vm/actor/src/builtin/sector.rs

```diff
@@ -4,6 +4,7 @@
 use fil_types::{RegisteredSealProof, StoragePower};
 
 /// Returns the minimum storage power required for each seal proof types.
+#[cfg(not(feature = "devnet"))]
```

Comment:
Just a note, you probably want to just have the feature functionality within the scope of the function, to be sure that there will be no differences in the future, but this is totally fine for this case